monkey/infection_monkey/exploit/agent_otp_provider.py

```diff
@@ -0,0 +1,11 @@
+from infection_monkey.island_api_client import IIslandAPIClient
+
+from .i_agent_otp_provider import IAgentOTPProvider
+
+
+class AgentOTPProvider(IAgentOTPProvider):
```

Comment:
Call this something like `IslandAPIAgentOTPProvider`.

Reply:
+1